server/games/game.py

```diff
@@ -269,17 +265,16 @@ def get_team_sets(self) -> List[Set[Player]]:
 
     async def wait_hosted(self, timeout: float):
         return await asyncio.wait_for(
-            asyncio.shield(self._is_hosted),
```

Comment:
Might want to keep the call to `shield`

Reply:
I tested it and it doesn't seem to matter. I think I added shield before because it prevented some exception from showing up in the logs when the server was shutting down, but that doesn't seem to happen with events.